src/models/transfer.py

```diff
@@ -171,7 +171,7 @@ def merge(self, other: Transfer) -> Transfer:
         if all(merge_requirements):
             return Transfer(
                 token=self.token,
-                solver=self.recipient,
+                recipient=self.recipient,
```

Comment:
so when merged, we sort by redirect target? Probably not an issue but could be a little confusing.

Reply:
Not really. When merged we sort by wherever the mutual receiver of the merged transfers is. Seems completely natural to me.